agent/acs/handler/acs_handler_test.go

```diff
@@ -519,7 +519,7 @@ func TestHandlerReconnectDelayForInactiveInstanceError(t *testing.T) {
 		mockWsClient.EXPECT().Connect().Do(func() {
 			reconnectDelay := time.Now().Sub(firstConnectionAttemptTime)
 			t.Logf("Delay between successive connections: %v", reconnectDelay)
-			if reconnectDelay < inactiveInstanceReconnectDelay {
+			if reconnectDelay+1 < inactiveInstanceReconnectDelay {
```

Comment:
On Window platform, we find intermittently that time.Now().Sub(...) can returns 199.9989ms  even after code waiting for time.NewTimer(200) ms